modules/client-configuration/pages/autoinstallation-methods.adoc

```diff
@@ -2,28 +2,16 @@
 = Introduction
 
 
+Kickstart and {ay} configuration files allow you to automate system installations for traditional clients.
+This is especially useful if you need to install a large number of clients.
 
+For {sle} clients, use {ay}.
+When you have created an {ay} file, you can upload it and manage it with {productname}.
 
-.Autoinstallation Types: {ay} and Kickstart
-[NOTE]
-====
-In the following section, {ay} and {ay} features apply for {sle} client systems only.
+For {rhel} clients, use Kickstart.
+Kickstart files are created, modified, and managed within the {productname} {webui}.
 
-For {rhela} systems, use Kickstart and Kickstart features.
-====
+The Cobbler installation server allows you to automate baremetal installations.
```

Comment:
```suggestion
The Cobbler installation server allows you to automate bare-metal installations.
```
(we spell it this way in the Web UI: Admin > Manager Config > Bare-metal Systems.)  In general, the term seems to be deprecated these days.  I think we should replace it with "physical system installations", or something similar.

Reply:
We probably should make it bare metal, according to this: https://books.google.com/ngrams/graph?content=baremetal%2Cbare-metal%2Cbare+metal&year_start=1900&year_end=2008&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cbaremetal%3B%2Cc0%3B.t1%3B%2Cbare%20-%20metal%3B%2Cc0%3B.t1%3B%2Cbare%20metal%3B%2Cc0

Although I agree with you on using a different term altogether, that's probably a different argument :wink: